Block-Logic/bl-rails-template
# 185714769 Tweak views and styling

#### What's this PR do?
- Tweaks views 

#### How should this be manually tested?
- Run test suite
- check all opt out views, including the form, deleting action, index table, email notification 
- click through all user forms, log in, sign up, password reminder etc.
- test cookie notification - accept cookies, refresh the page, delete cookies from browser, refresh again, etc.
- pay attention to how error messages, flash messages are displayed etc.

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/185714769)

#### Task completed checklist:

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
